docs/contributors/develop.md

```diff
@@ -13,3 +13,4 @@ The following resources offer additional information for developers who wish to
 * [Managing Packages](/docs/contributors/managing-packages.md).
 * [Gutenberg Release Process](/docs/contributors/release.md) - a checklist for the different type of releases for Gutenberg project.
 * [Localizing Gutenberg Plugin](/docs/contributors/localizing.md) - a guide on how to translate Gutenberg in your locale or language.
+* [React Native mobile Gutenberg](/docs/contributors/native-mobile.md) - a guide on the React Native based mobile Gutenberg editor.
```

Comment:
This file needs to be added to the config which manages developers documentation somehow similar to:
https://github.com/WordPress/gutenberg/blob/4cd15829241a25d979a00a59c9d33e35c27942eb/docs/manifest-devhub.json#L314-L319
and probably after these lines.

Reply:
Ah, didn't know, thanks for the direct pointer! Fixed with d1574fb.

Reply:
I guess it's another file where it needs to be added?

Reply:
How about: https://github.com/WordPress/gutenberg/blob/master/docs/toc.json?

/cc @chrisvanpatten

Reply:
I sorted it out.

Reply:
There is a related question about `manifest.json` file which wasn't updated here:
https://github.com/WordPress/gutenberg/pull/15254#issuecomment-551511172.